### ISSUE TITLE
Cannot get data only (payload includes the path)

### ISSUE THREAD

**Issue by laudaget** [semlette/nfc_in_flutter#11]
Is there a way to get the data without the path provided in payload? Couldn't figure out a way to cut away the path.

The content on my nfc chips are like 100001, 100002, 100003.. I would like to get '1000001' as my response, but the payload gives me '�en100001' instead.

Thank you in advance

**Comment by laudaget** [comment]
Thank you for your answer. 

I've tried several Apps from the playstore to write on the tags. Each app gives a diffrent suffix to my content if i read it out with your flutter plugin. But when i use other apps to read out the content, there is no suffix and i get the expectet output '100001'.

If it helps, my tag has the following specs: 
- XP NTAG 213 (NTAG213) - 180 Byte (NDEF: 137 Byte)
- ISO 14 443-2 A, ISO 14 443-3 A, NFC-Forum Typ 2, 13.56 MHz

you can find more about my tag: https://www.nfc-tag-shop.de/en/nfc-sticker/black-white-nfc-tags/103/nfc-sticker-pvc-on-metal-30-mm-ntag-213-180-byte-white?number=68236

**Comment by semlette** [comment]
Right, I've got some code brewing and I may be able to remove it.
I'll release a test version with the changes soon.

**Comment by semlette** [comment]
Could you try again using the `payload-data` branch?

- Try the example app in the `payload-data` branch
or
- Select the `payload-data` branch in your `pubspec.yaml` dependencies:
```yaml
dependencies:
  nfc_in_flutter:
    git:
      url: git://github.com/semlette/nfc_in_flutter.git
      ref: payload-data
```

**Comment by laudaget** [comment]
First of all, i really appreciate your efforts and time you put into this.

With the payload-data branch i was able to get the expectet output with `message.records.first.data`.

Because i am curious i would like to know what the prefix was. Was it the path or a variable to find the data?

Thank you very much.

**Comment by semlette** [comment]
After some reading I found out that it is part of the NDEF specification. Tags encoded with a "well known type" of "text", will have their (usually) first two characters be the language code of the text. By doing some magic, you can get the length of the language code and then remove it.

**Comment by semlette** [comment]
Closing this as I have (finally) released version 2.0.0, which adds the `.data` property for both iOS and Android.
